lib/std/time.zig

```diff
@@ -114,6 +114,13 @@ pub fn nanoTimestamp() i128 {
         return ns;
     }
 
+    if (builtin.os.tag == .uefi) {
+        var value: std.os.uefi.Time = undefined;
+        const status = std.os.uefi.system_table.runtime_services.getTime(&value, null);
+        assert(status == .Success);
+        return value.toEpoch();
```

Comment:
`toEpoch` returns seconds but this `nanoTimestamp` should return nanoseconds.

Reply:
Ok, that should be fixed now.

Reply:
Actually, I just realized UEFI time has nanoseconds so I'll change it to include that.